lib/features/feedback/widgets/custom_appbar.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg_provider/flutter_svg_provider.dart';
+import 'package:fluttercon/features/feedback/widgets/back_button.dart';
+
+class FeedbackCustomAppBar extends StatelessWidget {
+  const FeedbackCustomAppBar({super.key});
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 10, vertical: 10.0 * 3),
+      decoration: const BoxDecoration(
+        image: DecorationImage(
+          image: Svg('assets/images/feedback_appbar.svg'),
```

Comment:
As part of the app layout, the paths to the files are specified in `lib/common/utils/constants/app_assets.dart` and then referenced. 

Sample: `lib/common/widgets/app_bar/app_bar.dart:26`